.travis.yml

```diff
@@ -47,8 +47,12 @@ matrix:
     env: TOXENV=py36-django20-test
   - python: 3.6
     env: TOXENV=py36-django21-test
+  - python: 3.6
+    env: TOXENV=py36-django22-test
   - python: 3.6
     env: TOXENV=py36-flake
+  allow_failures:
+    - env: TOXENV=py36-django22-test
```

Comment:
Are there actual failures with Django 2.2? If there are, it would be better to fix them as part of this PR. If not, this seems like a liability, because I could forget to remove this later. Plus if some new patch introduces an incompatibility with Django 2.2 I'd rather know sooner than later.

Reply:
Yes, Django 2.2 build fails because Travis is using an old (not supported) version of SQLite:
> django.core.exceptions.ImproperlyConfigured: SQLite 3.8.3 or later is required (found 3.8.2).

Reply:
We can reinstall SQLite or we can wait until Travis will change its default version.

Reply:
Hmm, in that case lets leave it for now, hopefully Travis will update soon. Thanks for the info.

Reply:
I found another solution, i.e. switching this single build to xenial.